framework/src/preconditioners/PhysicsBasedPreconditioner.C

```diff
@@ -137,7 +137,7 @@ PhysicsBasedPreconditioner::PhysicsBasedPreconditioner(const InputParameters & p
   for (unsigned int var = 0; var < n_vars; var++)
     addSystem(var, off_diag[var], _pre_type[var]);
 
-  _nl.nonlinearSolver()->attach_preconditioner(this);
+  _nl.attachPreconditioner(this);
```

Comment:
@fdkong there are several places in Griffin that need to be patched like this here.

Reply:
OK, I will take a look